src/core/constants.mjs

```diff
@@ -81,7 +82,7 @@ export const GENESIS_KEY = process.env.GENESIS_KEY || '302e020100300506032b65700
 export const SYSTEM_ACCOUNTS = [[3, 100], [200, 349], [400, 750], [900, 1000]] // do account 0.0.2 last and outside the loop
 export const TREASURY_ACCOUNT = 2
 export const LOCAL_NODE_START_PORT = process.env.LOCAL_NODE_START_PORT || 30212
-export const ACCOUNT_CREATE_BATCH_SIZE = process.env.ACCOUNT_CREATE_BATCH_SIZE || 25
+export const ACCOUNT_CREATE_BATCH_SIZE = process.env.ACCOUNT_CREATE_BATCH_SIZE || 50
```

Comment:
the code is no longer using this as a 'batch size', so I don't think the name is appropriate anymore.

Reply:
Well, it still tries to process in a sequential 'group' or batch. So it still makes sense. We can reintroduce the 'parallel' update and see if that's stable.